tests/backend/checkmach/fail26.ml

```diff
@@ -0,0 +1,63 @@
+(* all functions fail the check when -disable-checkmach is passed. *)
```

Comment:
(Is it the same contents as
`fail25.ml`? if so, should we
have only one file processed
with different sets of flags?)

Reply:
yes, the files are identical but the little `gen_dune.ml` script doesn't support "same file with different flags" yet, as it would need a different output name.

Reply:
fixed.